generators/client/templates/angular/.eslintrc.json.ejs

```diff
@@ -35,7 +35,16 @@
   "parserOptions": {
     "ecmaVersion": 2018,
     "sourceType": "module",
-    "project": "./tsconfig.json"
+    "project": [
+      "./tsconfig.app.json",
+<%_ if (cypressTests) { _%>
+      "./<%= CLIENT_TEST_SRC_DIR %>cypress/tsconfig.json",
+<%_ } _%>
+<%_ if (protractorTests) { _%>
```

Comment:
Can't move files in their respective folders to ensure consistency?

Reply:
Protractor is deprecated and will be removed.
IMO we could even start to remove it before v8 due to not been maintained for so long. I doubt it’s usable.
Not worth to change anything at this point.